conf.yaml

```
@@ -1774,6 +1774,20 @@ contents:
               -
                 repo:   docs
                 path:   shared/attributes.asciidoc
+          - title:      Logstash and Kubernetes
+            prefix:     en/logstash-kubernetes
+            current:    main
+            branches:   [ {main: master} ]
+            index:      docs/k8sdocs/index.asciidoc
+            tags:       Logstash/Kubernetes
+            subject:    Logstash
+            sources:
+              -
+                repo:   logstash
+                path:   docs/k8dsdocs
```

Based on elastic/logstash#14457, I think this should be
```suggestion
                path:   docs/k8sdocs
```

---

Nice catch. Fixed. 👍